packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -289,6 +289,7 @@ export class Account {
    * @param {string} [options.token] Filter by token
    * @param {DfTxType} [options.txtype] Filter by transaction type. See DfTxType.
    * @param {number} [options.limit=100] Maximum number of records to return, 100 by default
+   * @param {number} [options.txn] Order in block, unlimited by default
    * @return {Promise<AccountHistory[]>}
```

Comment:
`AccountHistory` missing some fields for pool rewards. 
https://github.com/DeFiCh/ain/blob/acd6ce34181f54ab97a45733a6b830961ee1450f/src/masternodes/rpc_accounts.cpp#L65
might want to add a test for pool rewards and check. then adjust the `AccountHistory` accordingly.

Reply:
or may be create a separate ticket for that since it's out of the scope of this PR. Address it separately.

Reply:
Yes, it should be addresses separately.

Reply:
Created an issue for this https://github.com/DeFiCh/jellyfish/issues/1242